src/github.com/getlantern/flashlight/main/main.go

```diff
@@ -105,167 +71,49 @@ func main() {
 	client.ForceChainedProxyAddr = *forceProxyAddr
 	client.ForceAuthToken = *forceAuthToken
 
-	showui = !*headless
+	showui := !*headless
 
 	if showui {
-		runOnSystrayReady(_main)
+		runOnSystrayReady(_main(a))
 	} else {
 		log.Debug("Running headless")
-		_main()
+		_main(a)
```

Comment:
I think you have to actually call the returned func here @oxtoacart, as in `_main(a)()`